resources/helm/dask-gateway/templates/gateway/configmap.yaml

```diff
@@ -52,6 +52,8 @@ data:
             "%s:%s" % (image_name, image_tag) if image_tag else image_name
         )
 
+    c.KubeClusterConfig.imagePullSecrets = "{{ .Values.gateway.backend.imagePullSecrets }}"
```

Comment:
Is this the correct way to go about mapping the property?
I saw the other `backend` values were mapped here, so I went for it.

Reply:
- Look into if you see `KubeClusterConfig` which represents a Python class, have the configuration for `imagePullSecrets` then the left part of the assigment makes sense
- Note that `"{{ .Values.gateway.backend.imagePullSecrets }}"`` on the right side of `=` includes quotes, so the result will be `"[]"` which won't make sense - it should be a list, not a string.

Reply:
This could be made to make sense, but you need to define `image_pull_secrets` as well it seems, like image_pull_policy is defined already.

```
c.KubeClusterConfig.image_pull_secrets = {{ .Values.gateway.backend.imagePullSecrets }}
```

https://github.com/dask/dask-gateway/blob/e6ee014730991730ede59cbeb6ebaded27b0d22f/dask-gateway-server/dask_gateway_server/backends/kubernetes/backend.py#L51-L55